nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/Conv2D.java

```diff
@@ -84,6 +83,10 @@ public Conv2D(INDArray layerInput, INDArray weights, INDArray bias, Conv2DConfig
         this(layerInput, weights, bias, null, config);
     }
 
+    public Conv2D(@NonNull INDArray layerInput, @NonNull INDArray weights, INDArray bias, int wFormat, @NonNull Conv2DConfig conv2DConfig) {
+        this(layerInput, weights, bias, conv2DConfig);
+    }
```

Comment:
This constructor shouldn't be needed if weights format is in conv2dconfig?

Reply:
Fixed